### ISSUE TITLE
UI elements missing aria labels for accessibility

### ISSUE THREAD

**Issue by thienautran** [mckaywrigley/chatbot-ui#1217]
Throughout the UI, there are heavy use of icon buttons. While these contain tooltips. It would be great if buttons, and input dropdowns could have aria labels for better accessibility for screenreaders.

Most notable examples include
- Clickable icons in the ChatInput (@components/ui/chat-input) component are missing contextual labels. These include the send button, the add file button, and the settings
- chat settings button ChatSettings component (@/components/chat/chat-settings) is missing context on what the dropdown does. Consider adding an aria label that tells the user that "x model is being used, click to change settings for the selected model" or something similar.
- sidebar has no aria labels. Consider adding labels to each of the buttons in the sidebar to denote what they do.
- in the chat ui, buttons for each chat message do not have aria labels. Consider adding labels to the message buttons

**Comment by mckaywrigley** [comment]
Good PR opportunity.
